assignments/hackyourtemperature/package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "hack-your-temperature",
+  "version": "1.0.0",
+  "main": "server.js",
+  "type": "module",
+  "scripts": {
+    "test": "jest --coverage",
+    "test-watch": "jest --watchAll",
+    "start": "node --watch --env-file=.env server.js"
```

The `server.js` file has been rightfully moved under `src` folder so this needs to be updated to:

```suggestion
    "start": "node --watch --env-file=.env ./src/server.js"
```